docker/run.sh

```diff
@@ -9,8 +9,8 @@ export DBUS_SESSION_BUS_ADDRESS
 export DBUS_SESSION_BUS_PID
 
 LOGLEVEL=${LOGLEVEL:-"info"}
-SOTA_CLIENT="${SOTA_CLIENT_ADDR:-sota-client}:${SOTA_CLIENT_PORT:-9080}"
+SOTA_CLIENT="${SOTA_CLIENT_ADDR:-0.0.0.0}:${SOTA_CLIENT_PORT:-9080}"
 RVI="${RVI:-http://rvi-client:8901}"
 
 export RUST_LOG=${RUST_LOG:-"sota_client=$LOGLEVEL"}
-/bin/sota_client -c /var/sota/client.toml -r "$RVI" -e "$SOTA_CLIENT"
+/usr/bin/sota_client -c /var/sota/client.toml -r "$RVI" -e "$SOTA_CLIENT"
```

Comment:
Hi Jerry,

This patch is taken from the[ Yocto/OE recipe for building RVI SOTA client in GDP which I have recently contributed](http://git.projects.genivi.org/?p=meta-genivi-demo.git;a=commit;h=478d6625b9e9391e23445604b43c09ca76e12594). [Yesterday I was advised in the GENIVI mailing list to upstream this change because it makes general sense.](http://lists.genivi.org/pipermail/genivi-projects/2016-March/001827.html).

Best regards, Leon